src/components/event/NextEventsNavigation.js

```diff
@@ -8,7 +8,7 @@ const NextEventsNavigation = ({ additionalClass, prevText, nextText, pageContext
                 <a
```

Comment:
Please use Link component in all internal links

Reply:
fixed